### DISCUSSION TITLE
How do I install the license file?

### DISCUSSION THREAD

**Discussion by ralflukner** [iterate-ch/cyberduck#15826] [Q&A] [answered]
Where should I put the license file for the app purchased from the website? There's no registration option, and it doesn't seem to work. I have to locate an executable in the package.

**Comment by dkocher** [comment, marked as answer]
Make sure to save the registration key with the extension `.cyberducklicense` from your email program and then double click to open and register with _Cyberduck_.

**Comment by ralflukner** [comment]
Great, thank you.
—Dr. Lukner

**Comment by dkocher** [comment]
For Mountain Duck, make sure to save the registration key with the extension `.mountainducklicense` from your email program and then double click to open and register.

**Comment by ralflukner** [comment]
Thank you!  That worked.

————————————————
Ralf B. Lukner, MD PhD
Lukner Medical Clinic, PLLC
2545 Perryton Pkwy, Ste 31
Pampa, TX  79065
Office:  806-329-3050x20
Secure Fax:  806-905-6368
Web:  https://luknermed.com
Schedule a 25-minute meeting with me - Method 1 (Clockwise)
Schedule a 60-minute meeting with me - Method 2 (Calendly)
————————————————

**Comment by chriscara** [comment]
I received the email, saved it to my desktop, double clicked and a popup said "select an app to open this .cyberducklicense file".  "Cyberduck" was not an option and I have not been successful locating it.  What next?
